skmatter/datasets/descr/nice_dataset.rst

```diff
@@ -2,6 +2,7 @@
 
 NICE dataset
 ############
+
```

Comment:
I don't understand why these changes are here...

Reply:
They produced warnings in the doc building process. I removed them from this PR since I have to fix them in #172 anyway.